internal/backends/winit/renderer/skia.rs

```diff
@@ -28,15 +28,19 @@ mod cached_image;
 mod itemrenderer;
 mod textlayout;
 
+#[cfg(target_os = "macos")]
+mod metal_surface;
+
+#[cfg(target_family = "windows")]
+mod d3d_surface;
+
 cfg_if::cfg_if! {
     if #[cfg(skia_backend_opengl)] {
         mod opengl_surface;
         type DefaultSurface = opengl_surface::OpenGLSurface;
     } else if #[cfg(skia_backend_metal)] {
-        mod metal_surface;
         type DefaultSurface = metal_surface::MetalSurface;
```

Comment:
Does that means that this could be a runtime switch instead of a compile time switch?

Reply:
It's a bit more work to do that, but it's theoretically possible. I think we should only do that if a use-case arises.